### DISCUSSION TITLE
Azure Blob Implementation using Flexify.io

### DISCUSSION THREAD

**Discussion by Sonichigo** [nhost/hasura-storage#89] [General]
Recently while trying to implement the Azure Blob storage in Hasura-Storage, I found about the **Flexify.IO - Amazon S3 API for Azure Blob Storage** . It provides S3-compatible API for Azure Blob Storage enabling existing applications designed for Amazon Web Services to store data in Azure.  Flexify.IO adds S3 compatibility to an existing Azure Blob Storage. 

Majority of people work on S3 because of this friendliness, so i guess not complete shifting is necessary, if needed Azure Blob Storge Service, Flexify.IO can be used, as in that case no changes would be needed to ``@nhost/hasura-storage`` codebase too as far as i know.

**Comment by dbarrosop** [comment]
Yeah, that could be an option, I think adding support "natively" for azure would also be good if someone wanted to implement it. Altneratively, you can do [this](https://cloudblogs.microsoft.com/opensource/2017/11/09/s3cmd-amazon-s3-compatible-apps-azure-storage/) as well.

**Reply by Sonichigo** [reply]
I thought of implementing Azure support but with their new ``sdk`` for blob, but was a bit much for me with my current experience.
